gensrc/java/com/starrocks/udf/UDFHelper.java

```diff
@@ -0,0 +1,257 @@
+package com.starrocks.udf;
+
+import java.nio.ByteBuffer;
+import java.nio.IntBuffer;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+
+public class UDFHelper {
+    public static final int TYPE_BOOLEAN = 2;
+    public static final int TYPE_TINYINT = 3;
+    public static final int TYPE_SMALLINT = 4;
+    public static final int TYPE_INT = 5;
+    public static final int TYPE_BIGINT = 6;
+    public static final int TYPE_FLOAT = 8;
+    public static final int TYPE_DOUBLE = 9;
+    public static final int TYPE_VARCHAR = 10;
+    public static final int TYPE_DATETIME = 12;
+    public static final int TYPE_ARRAY = 15;
+
+    // create boxed array
+    public static Object[] createBoxedArray(int type, int numRows, boolean nullable, ByteBuffer... buffer) {
```

Comment:
When and why we need create boxed array?

Reply:
There is an Array parameter as input in batchUpdate